src/app/tasks/task-list.model.ts

```diff
@@ -0,0 +1,7 @@
+import { Task } from "./task.model";
+
+export class TaskList {
+    constructor(
```

Comment:
Use types or interfaces instead of classes

Reply:
still here, instead of defining classes just define a type or an interface for example 
export type Task = {
id: string,
name: string,
createOn: string
etc
}